### PR TITLE
create ecom home_page

### PR DESCRIPTION
**Pull request by git-0r**
# Created ecom home page screen
ecom2
ecom1

### CODE REVIEW

**Review by aamrapaliwandhre — APPROVED** [review]

**Review by harshparab — APPROVED** [review]
looks good

File: index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                                    1.06-1.06a5.5 5.5 0 0 0 0-7.78z">
+                                </path>
+                            </svg>
+                        </p>
+                    </div>
+                </div>
+                <div class="card card-vertical">
+                    <img class="card-img"
+                        src="https://res.cloudinary.com/clouduser/image/upload/c_scale,w_198,h_200/v1645367388/download_lym0ig.jpg"
+                        alt="">
```
Comment (Naman-Saxena1) [inline index.html]: alt attribute is empty

File: index.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+                </div>
+            </div>
+        </div>
+        <div class="most-selling">
+            <header class="heading-lg">Most selling products</header>
+            <div class="product-card-container d-flex flex-justify-evenly">
+                <div class="card card-vertical">
+                    <img class="card-img"
+                        src="https://res.cloudinary.com/clouduser/image/upload/c_scale,w_198/v1645367117/download_hm5h9k.jpg"
+                        alt="">
```
Comment (Naman-Saxena1) [inline index.html]: Same here

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (git-0r) [reply index.html]: done

**Comment by varun1233** [comment]
UI Suggestion: Use cursor : pointer on icons in header and footer

**Review by Sahil25061999 — APPROVED** [review]
UI suggestion reduce the spacing for the nav buttons and improve the visibility for the wishlist button on the cards. Also add some hover effects  for the shop now button.

**Pull request merged**